src/ray/common/ray_config_def.h

```diff
@@ -248,7 +248,7 @@ RAY_CONFIG(int32_t, maximum_profile_table_rows_count, 10 * 1000)
 RAY_CONFIG(uint32_t, object_store_get_max_ids_to_print_in_warning, 20)
 
 /// Number of threads used by rpc server in gcs server.
-RAY_CONFIG(uint32_t, gcs_server_rpc_server_thread_num, 1)
+RAY_CONFIG(uint32_t, gcs_server_rpc_server_thread_num, 4)
```

Comment:
Would be great to not have to change this (doesn't it break k8s?) Should be a separate PR if we do decide to do it.

Reply:
it doesn't break k8s. we fixed that in the test